packages/logo-react/documentation/LogoExample.tsx

```diff
@@ -1,9 +1,11 @@
 import React from "react";
 import { ExampleComponentProps } from "@fremtind/jkl-portal-components";
 import { Logo } from "../src";
+import "./LogoExample.scss";
 
 export const LogoExample = ({ boolValues }: ExampleComponentProps) => (
     <Logo
+        className="logo-example"
```

Comment:
Kan evt legge inn height og width som props på svgen, så kan man dynamisk sende det inn isteden for å sette klasse rundt :)

Reply:
Det er også en kul løsning! Tenker vi fint kan ha støtte for begge deler 😊 Jeg kjører inn dette nå (sånn at eksempelet faktisk fungerer), så kan vi legge til støtte for props.